system/react/src/components/Toggle.ts

```diff
@@ -0,0 +1,121 @@
+import styled from '@emotion/styled';
+
+export const classySelector = 'input[type="checkbox"].toggle';
+
+export const Toggle = styled.input<{ type?: 'checkbox' }>`
```

Comment:
```suggestion
export const Toggle = styled.input<{ type: 'checkbox' }>`
```

This styling is only applicable to a checkbox so we should enforce it.

Reply:
fixed